src/components/TimeQuerier/index.tsx

```diff
@@ -13,28 +13,35 @@ import {
   Alert,
   AlertIcon,
 } from "@chakra-ui/react";
-import moment from "moment";
-import Datetime from "react-datetime";
 import constants from "../../utils/constants";
 import { useTimeQuerierStore } from "../../store/timeQuerier";
+import dayjs from "dayjs";
+import DateFnsUtils from "@date-io/date-fns";
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from "@material-ui/pickers";
 
 const TimeQuerier: React.FC = () => {
   const styles = useStyleConfig("DateTime", {});
-  const { selectedStartTimestamp, selectedEndTimestamp, changeTimeQuerier } =
-    useTimeQuerierStore();
+  const {
+    selectedStartTimestamp,
+    selectedEndTimestamp,
+    changeTimeQuerier,
+  } = useTimeQuerierStore();
   const { defaultStepValue, minStepValue, dateFormat, timeFormat } = constants;
-  const [startTime, setStartTime] = useState(moment(selectedStartTimestamp));
-  const [endTime, setEndTime] = useState(moment(selectedEndTimestamp));
+  const [startTime, setStartTime] = useState(dayjs(selectedStartTimestamp));
+  const [endTime, setEndTime] = useState(dayjs(selectedEndTimestamp));
```

Comment:
I'd say keep the start and end times in the redux store (and persist them too) so that it is maintained across refreshes (mostly people do not like dates getting reset)